h2o-r/h2o-package/R/models.R

```diff
@@ -3605,6 +3605,12 @@ h2o.getModelTree <- function(model, tree_number, tree_class = NA) {
 
   res$thresholds[is.nan(res$thresholds)] <- NA
 
+  if(res$left_children[1] == -1 && res$right_children == -1){
```

Comment:
This has to be looked into. We've seen an issue because of this with @honzasterba. Java backend was in an infinite loop. I would rather mimic what xgboost-predictor is doing.

It is not possible for a node to have just one child node.

Reply:
@michalkurka What exactly needs to be looked into ? Why is this happening ? I don't understand what you're referring to.

Reply:
Checks if the root node has no children. Just `res$left_children[1] == -1` should be sufficient, as I can't think of any anomaly causing the root node to have just one child as a leaf node, yet I wanted to be 100% sure only this one case where root node is the only node in the graph is fixed.

Didn't want to make the class structure any more complicated, so I fixed it by replacing the invalid types backend's response.